tests/mongo-go-driver.yml

```diff
@@ -67,14 +71,35 @@ results:
       - go.mongodb.org/mongo-driver/x/mongo/driver/topology/TestCMAPSpec/pool-checkin-destroy-closed.json
       - go.mongodb.org/mongo-driver/x/mongo/driver/topology/TestCMAPSpec/pool-close-destroy-conns.json
 
+      # event ordering incompatible with load-balancer SDAM spec test
+      - go.mongodb.org/mongo-driver/x/mongo/driver/topology/TestCMAPSpec/pool-create-min-size-error.json
+
+      # test requires that connections established by minPoolSize
+      # are immediately used to satisfy check-out requests 
+      - go.mongodb.org/mongo-driver/x/mongo/driver/topology/TestCMAPSpec/pool-checkout-minPoolSize-connection-maxConnecting.json
+
+      # test requires a checked-in connections cannot satisfy a check-out waiting on a new connection
+      - go.mongodb.org/mongo-driver/x/mongo/driver/topology/TestCMAPSpec/pool-checkout-returned-connection-maxConnecting.json
+
+      # skip if CRYPT_SHARED_LIB_PATH is not set
+      - go.mongodb.org/mongo-driver/mongo/TestClient/mongocryptd_or_crypt_shared
+
+      # skip if no compressor specified
+      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestCompression
+
+      # skip because authentication is required
+      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestSCRAM
+
+      - go.mongodb.org/mongo-driver/x/mongo/driver/integration/TestInsert
```

Comment:
I'm not sure how to comment that line.
The test just [skips on the first line of it](https://github.com/mongodb/mongo-go-driver/blob/master/x/mongo/driver/integration/insert_test.go#L20)

Reply:
I think that could be something like "Always skipped"